media/CircleGraph/index.js

```diff
@@ -97,6 +98,8 @@ host.BrowserHost = class {
       this._mode = viewMode.selector;
     } else if (__viewMode === "visq") {
       this._mode = viewMode.visq;
+    } else if (__viewMode === "visqselr") {
```

Comment:
please split PRs. I don't want `media/CircleGpaph` codes to be mixed with others.

Reply:
Sorry... ok.